eslint.config.mjs

```diff
@@ -64,7 +67,16 @@ export default tseslint.config(
       "@typescript-eslint/no-non-null-asserted-optional-chain": "warn",
       "@typescript-eslint/ban-types": "off",
       "@typescript-eslint/no-explicit-any": "off",
-      "@typescript-eslint/no-unused-vars": "off",
+      "@typescript-eslint/no-unused-vars": [
+        "error",
```

Comment:
This config is a bit more forgiving, because it:
- allows to leave `catch (error)` as is
- checks only local variables, and not globals (e.g. on `window`)
- checks unused arguments in functions, methods after last used (e.g. `unusedArg1, arg2, arg3, unusedArg4` where last used was `arg3`, only `unusedArg4` will raise error)
- ignores unused variables and arguments that start with `_` which is a common pattern